looker/resource_looker_user_attribute_group_test.go

```diff
@@ -0,0 +1,102 @@
+package looker
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	sdk "github.com/looker-open-source/sdk-codegen/go/sdk/v4"
+	"github.com/resolutionlife/terraform-provider-looker/internal/conv"
+)
+
+func init() {
+	// Add a sweeper to remove user attributes and groups that have names starting with `test_acc`.
+	resource.AddTestSweepers("looker_user_attribute_group", &resource.Sweeper{
+		Name: "looker_user_attribute_group",
+		F: func(r string) error {
+			c, err := newTestLookerSDK()
+			if err != nil {
+				return err
+			}
+
+			userAttrs, err := c.AllUserAttributes(sdk.RequestAllBoardSections{
+				Fields: conv.P(""),
+				Sorts:  conv.P(""),
+			}, nil)
+			if err != nil {
+				return err
+			}
+
+			filteredUserAttrs := make([]sdk.UserAttribute, 0, len(userAttrs))
+			for _, userAttr := range userAttrs {
+				if strings.HasPrefix(userAttr.Name, "test_acc_") {
+					filteredUserAttrs = append(filteredUserAttrs, userAttr)
+				}
+			}
+
+			for _, filteredUserAttr := range filteredUserAttrs {
+				if _, err := c.DeleteUserAttribute(*filteredUserAttr.Id, nil); err != nil {
+					return err
+				}
+			}
+
+			groups, err := c.SearchGroups(sdk.RequestSearchGroups{
+				Name: conv.PString("test-acc%"),
+			}, nil)
+			if err != nil {
+				return err
+			}
+
+			for _, g := range groups {
+				if _, err := c.DeleteGroup(*g.Id, nil); err != nil {
+					return err
+				}
+			}
```

Comment:
Same comment as https://github.com/resolutionlife/terraform-provider-looker/pull/39#discussion_r979752213.
Maybe we could clean up the sweepers as part of another PR?